apps/packages-playground/src/index.css

```diff
@@ -0,0 +1,34 @@
+:root {
+  font-family: Inter, system-ui, Avenir, Helvetica, Arial, sans-serif;
+  line-height: 1.5;
+  font-weight: 400;
+
+  color-scheme: light dark;
+  color: rgba(255, 255, 255, 0.87);
+  background-color: #242424;
```

Comment:
styles should use design tokens.

Reply:
These styles are part of the Vite boilerplate code. removing it as we don't need it.